RXPatchLib/WebPatchSource.cs

```diff
@@ -194,6 +194,8 @@ private void UnlockDownload()
                                 cancellationToken.ThrowIfCancellationRequested();
                                 AXDebug.AxDebuggerHandler.Instance.RemoveDownload(guid);
 
+                                if (thisPatchServer != null) thisPatchServer.HasErrored = true;

```

Comment:
We probably shouldn't do this here. If too many WebExceptions pile up, TooManyRetriesException will bubble up and do this already.